servers/src/main/java/tachyon/master/file/meta/PersistenceState.java

```diff
@@ -21,5 +21,6 @@
 public enum PersistenceState {
   NOT_PERSISTED, // file not persisted in the under FS
   IN_PROGRESS, // used for async persistence, the async persistence is scheduled and in progress
-  PERSISTED // the file is persisted in the under FS
+  PERSISTED, // the file is persisted in the under FS
+  LOST // the file is lost but not persisted in the under FS
```

Comment:
Discussed offline,  `LOST` indicates a file is not in Tachyon any more and can never be persisted.
`NOT_PERSISTED` means the file in Tachyon only.